src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -778,14 +778,13 @@ async def _genomic_to_tx_segment(
                 )
 
         if genomic_ac:
-            genomic_ac_validation = await self.uta_db.validate_genomic_ac(genomic_ac)
-            if not genomic_ac_validation:
+            grch38_ac = await self.uta_db.get_newest_assembly_ac(genomic_ac)
+            if grch38_ac:
+                genomic_ac = grch38_ac[0]
```

Comment:
Is it worth adding something in the docstrings (for both private + public method) about version will be ignored for GRCh37?